### DISCUSSION TITLE
Introducing buncors, a bundled bunrest compatible package

### DISCUSSION THREAD

**Discussion by lewisjr** [oven-sh/bun#6742] [Show and tell]
![buncors promo](https://ibin.co/w800/7fBFa3a4a06T.png)

I've made a [package](https://www.npmjs.com/package/buncors) to help with handling cors on a bun server made using [bunrest](https://github.com/lau1944/bunrest). Whether it's a normal REST request or a preflight request, it will adapt and handle the requests as required.

**Comment by vitalspace** [comment]
well another http framework
